redfish_client/connector.py

```diff
@@ -53,3 +150,6 @@ def patch(self, path, payload=None):
 
     def delete(self, path):
         return self._request("DELETE", path)
+
+    def reset(self, _path=None):
+        pass
```

Comment:
This reset is here because of our double-connector classing maneuver. If you decide to unify it into a single class as suggested in my comment above, you can safely remove it.